### DISCUSSION TITLE
Zen77 800LR Will Not Include Insecurely

### DISCUSSION THREAD

**Discussion by ioguy** [zwave-js/zwave-js#6810] [Request Support / Investigate Issue] [answered]
SiLabs 700 USB Controller 7.21.2.

I have tried 3 different Zen77 800LRs.
Each I try to include insecurely and I get the following errors in the log:

2024-05-05T22:59:09.499Z CNTRLR   Starting inclusion process with strategy Insecure...
2024-05-05T22:59:09.517Z CNTRLR   The controller is now ready to add nodes
2024-05-05 18:59:09.518 INFO Z-WAVE: Controller status: Non-secure inclusion started
2024-05-05 18:59:09.521 INFO Z-WAVE: Success zwave api call startInclusion true
2024-05-05 18:59:16.780 INFO APP: GET /health/zwave 301 1.188 ms - 191
2024-05-05T22:59:18.456Z DRIVER   Dropping message because it could not be deserialized: The command class unkno
                                  wn (0xc7) is not implemented (ZW0303)
2024-05-05T22:59:18.896Z DRIVER   Dropping message because it could not be deserialized: The command class unkno
                                  wn (0xc7) is not implemented (ZW0303)
2024-05-05T22:59:18.939Z DRIVER   Dropping message because it could not be deserialized: The command class unkno
                                  wn (0xc7) is not implemented (ZW0303)
2024-05-05T22:59:18.955Z DRIVER   Dropping message because it could not be deserialized: The command class unkno
                                  wn (0xc7) is not implemented (ZW0303)
2024-05-05T22:59:23.394Z CNTRLR   handling add node request (status = NodeFound)
2024-05-05T22:59:26.598Z CNTRLR   handling add node request (status = Failed)
2024-05-05T22:59:26.599Z CNTRLR   Adding the node failed
2024-05-05 18:59:26.600 INFO Z-WAVE: Controller status: Inclusion failed
2024-05-05T22:59:26.601Z CNTRLR   stopping inclusion process...
2024-05-05T22:59:26.620Z CNTRLR   The inclusion process was stopped
2024-05-05 18:59:26.621 INFO Z-WAVE: Controller status: Inclusion stopped

Any ideas?  I haven't tried securely but also don't really want to have them secure.  All my devices with the exception of locks are insecure.

Also something internal to the USB controller may be happening because I just added a Zen14 800LR and it showed up as node 71 when my last added node was 62.

**Comment by AlCalzone** [comment]
> handling add node request (status = Failed)

This is typically an indication of either bad connectivity, or that the controller and/or the end device needs to be power cycled.

**Reply by ioguy** [reply]
I tried power cycling both the controller and the end devices (it's actually 3 separate Zen77s that won't include) and it didn't help.  I wired a plug onto one of the Zen77s and moved it within 10' of the controller, still no luck.

**Reply by AlCalzone** [reply]
Maybe try excluding the ZEN77 before inclusion, even though it is not included. Or try factory resetting it.
Unfortunately the error message doesn't give us much to work with, as this part of the communication is only between the controller and the device, and we never see any details.

**Reply by ioguy** [reply]
Tried exclusion and factory reset with no luck.  They just won't accept an insecure inclusion.

I was able to include them using SmartStart and LR.  Now I have range issues with it in the shed (75' away from the controller).  Maybe it's time to try my 800 series controller to see if that improves.

I tried adding them regular through SmartStart and the log always showed:
> Dropping message because it could not be deserialized: The command class unknown (0xc7) is not implemented (ZW0303)

**Reply by ioguy** [reply]
So a little update with my playing...
My 800 stick behaved the same when trying to pairs the Zen77s.
I moved my sticks (700 and 800) to PC Controller and the Zen77s paired immediately without any issue.  Does this point to a Z-Wave JS issue?

**Reply by AlCalzone** [reply]
Again, `status = Failed` is between the stick and the end device. It basically tells us "sorry, couldn't include that device".

I assume by moving to PC controller, you physically moved the stick to a different location, and probably a different USB port?
-> https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting

**Reply by ioguy** [reply]
Bizarre.  Yes, different USB port/computer.  Same location.  It's odd that other devices will pair fine, just not the Zen77s.

Question.  If I copy my security keys from Z-Wave JS into PC Controller and include the devices from within PC Controller, will Z-Wave JS pick them up when I move the stick back to HA?  I would just re-interview to get all the details into JS?

**Reply by AlCalzone** [reply, marked as answer]
>  If I copy my security keys from Z-Wave JS into PC Controller and include the devices from within PC Controller, will Z-Wave JS pick them up when I move the stick back to HA? I would just re-interview to get all the details into JS?

Yes.

> Yes, different USB port/computer

I'm able to use my Z-Wave controllers just fine on my PC when I plug them into the USB ports at the front panel. On my laptop I have to use an extension cable [like we recommend](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting) anyways, or nothing works.

**Reply by ioguy** [reply]
OK, thanks for your help, appreciate it.

I already use an extension cable on HA, it's run on an Intel NUC.  I'll do the inclusion on PC Controller (separate computer) and then re-interview.
